docker/images/superset/Dockerfile

```diff
@@ -0,0 +1,112 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+ARG NODE_VERSION=latest
+ARG PYTHON_VERSION=latest
+
+# --- Build assets with NodeJS
+
+FROM node:${NODE_VERSION} AS build
+
+# Superset version to build
+ARG SUPERSET_VERSION=master
```

Comment:
maybe set this to the latest released version or use ENV variable

Reply:
There is an ENV variable. Please see `README.md` for that.